File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int maxCount = 0;
        int n = nums.size();
        int cnt = 0;
        for(int i = 0; i<n; i++){
            if(nums[i]==1){
                cnt++;
            }else{
                cnt = 0;
            }
            maxCount = max(maxCount, cnt);
        }
        return maxCount;
    }
};

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int c = 0;
        int maxC = 0;
        if (nums.length == 1) {
            return nums[0];
        }
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] == 1) {
                c++;
            } else {
                c = 0;
            }
            maxC = Math.max(c, maxC);
        }
        return maxC;
    }
}